applogic/src/api/conversation_list_cubit.rs

```diff
@@ -0,0 +1,157 @@
+// SPDX-FileCopyrightText: 2024 Phoenix R&D GmbH <hello@phnx.im>
+//
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+use flutter_rust_bridge::frb;
+use log::{error, info};
+use phnxcoreclient::clients::CoreUser;
+use phnxcoreclient::ConversationId;
+use tokio::sync::{broadcast, watch};
+use tokio_util::sync::CancellationToken;
+
+use crate::api::user::User;
+use crate::util::{spawn_from_sync, Cubit, CubitCore};
+use crate::StreamSink;
+
+use super::messages::{FetchedMessages, FetchedMessagesReceiver};
+use super::types::UiConversationDetails;
+use super::user::user_cubit::UserCubitBase;
```

Comment:
Is there a good reason not to merge the imports? We should agree on a standard. So far we have used merged imports, but I'm open to good arguments.

Reply:
No particular reason except my global config is like that:

```lua
imports = {
    granularity = {
        group = "module",
    },
},
```

Most likely we should configure this in `.rustfmt.toml`

Reply:
Unfortunately, https://rust-lang.github.io/rustfmt/?version=v1.6.0&search=imports#imports_granularity is not yet stable.